### PR TITLE
Add doc for fleetobservability feature spec

### PR DESCRIPTION
**Pull request by JeffLuoo**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/14808.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
gkehub: added doc for new field `fleetobservability` in `google_gke_hub_feature` (beta)
```

### CODE REVIEW

**Comment by JeffLuoo** [comment]
cc: @couragej for the review of the change.

File: mmv1/third_party/terraform/website/docs/r/gke_hub_feature.html.markdown [hunk 1]
```diff
@@ -71,6 +71,66 @@ resource "google_gke_hub_feature" "feature" {
...
+      logging_config {
+        default_config {
+          mode = "COPY"
+        }
+      }
+    }
+  }
+  provider = google-beta
+}
+```
```
Comment (courageJ) [inline mmv1/third_party/terraform/website/docs/r/gke_hub_feature.html.markdown:91]: Any examples to unset the feature?

Comment (JeffLuoo) [reply mmv1/third_party/terraform/website/docs/r/gke_hub_feature.html.markdown:91]: Resolve to move our discussion to b/286127157. Will update the doc once we have the update.

**Comment by JeffLuoo** [comment]
@sahsagar-google will take over the documentation change for the `fleetobservability` field.

**Pull request closed without merging**